snippets/javascript/class-object.md

```diff
@@ -0,0 +1,39 @@
+# Text encryptor
+
+You are making a text encryptor. It should take multiple words and output a
+combined version, where each word is separated by a dollar sign `$`.
+
+For example, for the words "hello", "how", "are", "you", the output should be
+`"$hello$how$are$you$"`.
+
+```js
+class Add {
+  constructor(...words) {
```

Comment:
here also add a comment explaining what the constructor and the spread syntax(...).

Reply:
`Spread syntax (...)` allows an iterable such as an array expression or string to be expanded in places where zero or more arguments (for function calls) or elements (for array literals) are expected, or an object expression to be expanded in places where zero or more key-value pairs (for object literals) are expected.

The `constructor` method is a special method for creating and initializing an `object` created with a `class`.